### ISSUE TITLE
Missing code issues in the project

### ISSUE THREAD

**Issue by divanoresia** [zhangweifeng1218/Adaptive_RF_Fingerprinting#3]
Hello, after reading your article, I feel very inspired, but I find that the data and related directories seem to be missing in the code of the paper. Could you please send the complete code to divanoresiakt@gmail.com?Thank you most sincerely.

**Comment by zhangweifeng1218** [comment]
Thank you for your attention. The data used in our paper was collected in collaboration with&nbsp;China Electronics Technology Group Corporation （CETC）. Unfortunately, due to our contract with them, we are not authorized to release these datasets.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zhangweifeng1218/Adaptive_RF_Fingerprinting"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年12月6日(星期三) 下午2:49
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[zhangweifeng1218/Adaptive_RF_Fingerprinting] Missing code issues in the project (Issue #3)





 
Hello, after reading your article, I feel very inspired, but I find that the data and related directories seem to be missing in the code of the paper. Could you please send the complete code to ***@***.***?Thank you most sincerely.

**Comment by divanoresia** [comment]
Thank you for your response! Thank you for your great contribution to the field of RF fingerprinting!

I am currently reproducing the models from your paper "Adaptive RF Fingerprints Fusion via Dual Attention Convolutions" and the comparison models mentioned in the article based on your github code. There are ARFNet, AlexNet-1D, ResNet-50-1D, CVCNN and BP-CNN-2D.

Since I am using discrete samples of transmit signal amplitude as my source data, I uniformly changed the number of channels in the convolutional layer in your open source code from 2 to 1. I am now experimenting with 5 samples per transmitter from my source data to train the ARFNet, AlexNet-1D, ResNet-50-1D, CVCNN, BP-CNN-2D models. 2D models, but the models were not trained well. Could you provide the hyperparameters and loss functions that worked best for you to train the six models ARFNet, AlexNet-1D, ResNet-50-1D, CVCNN, and BP-CNN-2D at that time? I didn't find the best hyperparameter settings and related information in your article.
Thank you most sincerely!
Wishing you all the best！



At 2023-12-07 12:50:54, "Weifeng Zhang" ***@***.***> wrote:

Thank you for your attention. The data used in our paper was collected in collaboration with&nbsp;China Electronics Technology Group Corporation （CETC）. Unfortunately, due to our contract with them, we are not authorized to release these datasets.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "zhangweifeng1218/Adaptive_RF_Fingerprinting" ***@***.***&gt;;
发送时间:&nbsp;2023年12月6日(星期三) 下午2:49
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[zhangweifeng1218/Adaptive_RF_Fingerprinting] Missing code issues in the project (Issue #3)






Hello, after reading your article, I feel very inspired, but I find that the data and related directories seem to be missing in the code of the paper. Could you please send the complete code to ***@***.***?Thank you most sincerely.
